tests/subsys/input/api/testcase.yaml

```diff
@@ -9,6 +9,11 @@ tests:
   input.api.thread:
     extra_configs:
       - CONFIG_INPUT_MODE_THREAD=y
+      # The test for being unable to push another message into full queue
+      # would fail if there are another CPU processing the callbacks,
+      # thus making the queue non-full. So limit this to 1 CPU only so
+      # the test assumptions can be fulfilled.
```

Comment:
The comment seems a bit mangled up? That or I can't read anymore. Or both.

Reply:
There is a check to see if you cannot push another message anymore. However, if there is another CPU starting consuming messages, the queue would not be full at that point, and that check will fail. Let me see about changing the wording.

Reply:
ok so it was the "I can't read anymore" option, thanks :-)